code-challenges.test.js

```diff
@@ -38,12 +47,21 @@ const letterA = "a"
 const arrayOfWords2 = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach"]
 const letterE = "e"
 // Expected output: ["Cherry", "Blueberry", "Peach"]
-
-
-
+describe('letterFinder', () => {
+    it('takes in array of words and single letter, returns all words that contain particular letter', () => {
+        expect(letterFinder(arrayOfWords1)).toContain(["Apple", "Banana", "Orange"]),
+        expect(letterFinder(arrayOfWords2)).toContain(["Mango", "Cherry", "Apricot", "Blueberry", "Peach"])
+        });  
+});
 // b) Create the function that makes the test pass.
-
-
+const letterFinder = (array, letter) => {
+        let newArray = array.filter(value => {
+          // new array will be the filtered array
+      return value.includes(letter.toLowerCase()) || value.includes(letter.toUpperCase())
+      // returns value irregardless of case
+    })
+      return newArray
+}
```

Comment:
Nice work. If you find yourself creating a variable just to return it you can shorten your code and just return the logic. `return array.filter(value.......`